cpp/src/arrow/ipc/metadata_internal.cc

```diff
@@ -236,7 +236,8 @@ static inline TimeUnit::type FromFlatbufferUnit(flatbuf::TimeUnit unit) {
   return TimeUnit::SECOND;
 }
 
-constexpr int32_t kDecimalBitWidth = 128;
+constexpr int32_t kDecimalBitWidth128 = 128;
+constexpr int32_t kDecimalBitWidth256 = 256;
```

Comment:
I'm frankly not sure those pleonasmic constants are useful.

Reply:
removing them.